player.lua

```diff
@@ -70,7 +72,7 @@ function Player:collide()
 				jump:play()
 			end
 		end
-		if col.other.name == 'player' then
+		if col.other.name == 'player' and col.other.alive then
```

Comment:
If you want to avoid colliding with a dead player, [remove its body from the world](https://github.com/kikito/bump.lua) (look into world:remove)